README.md

```diff
@@ -126,47 +88,84 @@ kustomize build 'github.com/kubernetes-sigs/descheduler/kubernetes/deployment?re
 | `nodeSelector` | `nil` | limiting the nodes which are processed |
 | `evictLocalStoragePods` | `false` | allows eviction of pods with local storage |
 | `evictSystemCriticalPods` | `false` | [Warning: Will evict Kubernetes system pods] allows eviction of pods with any priority, including system pods like kube-dns |
 | `ignorePvcPods` | `false` | set whether PVC pods should be evicted or ignored |
-| `maxNoOfPodsToEvictPerNode` | `nil` | maximum number of pods evicted from each node (summed through all strategies) |
-| `maxNoOfPodsToEvictPerNamespace` | `nil` | maximum number of pods evicted from each namespace (summed through all strategies) |
 | `evictFailedBarePods` | `false` | allow eviction of pods without owner references and in failed phase |
+|`labelSelector`|(see [label filtering](#label-filtering))|
+|`priorityThreshold`|int (see [priority filtering](#priority-filtering))|
+|`thresholdPriorityClassName`|string (see [priority filtering](#priority-filtering))|
+|`nodeFit`|bool (see [node fit filtering](#node-fit-filtering))|
+
+### Example policy
+
+As part of the policy, you will start deciding which top level configuration to use, then which Evictor plugin to use (if you have your own, the Default Evictor if not), followed by deciding the configuration passed to the Evictor Plugin. After that you will enable/disable eviction strategies plugins and configure them properly.
 
-As part of the policy, the parameters associated with each strategy can be configured.
-See each strategy for details on available parameters.
+See each eviction strategy plugin section for details on available parameters.
 
 **Policy:**
 
 ```yaml
-apiVersion: "descheduler/v1alpha1"
+apiVersion: "descheduler/v1alpha2"
 kind: "DeschedulerPolicy"
-nodeSelector: prod=dev
-evictFailedBarePods: false
-evictLocalStoragePods: true
-evictSystemCriticalPods: true
-maxNoOfPodsToEvictPerNode: 40
-ignorePvcPods: false
-strategies:
-  ...
+nodeSelector: "node1" # you don't need to set this, if not set all will be processed
+maxNoOfPodsToEvictPerNode: 5000 # you don't need to set this, unlimited if not set
+maxNoOfPodsToEvictPerNamespace: 5000 # you don't need to set this, unlimited if not set
+profiles:
+  - name: ProfileName
+    pluginConfig:
+    - name: "DefaultEvictor"
+      args:
+        evictSystemCriticalPods: true
+        evictFailedBarePods: true
+        evictLocalStoragePods: true
+        nodeFit: true
+    plugins:
+      evict:
+        enabled:
+          - "DefaultEvictor"
+      deschedule:
+        enabled:
+          - ...
+      balance:
+        enabled:
+          - ...
+      [...]
 ```
 
 The following diagram provides a visualization of most of the strategies to help
 categorize how strategies fit together.
 
 ![Strategies diagram](strategies_diagram.png)
 
+Next sections will go over different eviction strategy plugins. Right now we group them by being Deschedule plugins or Balance plugins (more types could be added).
```

Comment:
"being Deschedule plugins" => "implementing Deschedule extension point"
similar for the Balance

Reply:
Added the table and expanded this explanation